ember-phone-input/src/components/phone-input.ts

```diff
@@ -301,10 +345,10 @@ export default class PhoneInputComponent extends Component {
     this.update(this.number, this._metaData(_iti));
   }
 
-  _metaData(iti) {
+  _metaData(iti: intlTelInput.Plugin | null): MetaData {
```

Comment:
If the point of having the `_` prefix is to signal a `private` member, I think we should explicitly mark this as `private`. In the same fashion as: https://github.com/qonto/ember-amount-input/pull/610/files#diff-2943b055d0c8d58dd94ec0223afdaed6a960beeddc0466a35baa8c3f2dc070e2R170

What do you think?

Reply:
Fixed